src/routes/Account/Overview/Transactions/Transactions.tsx

```diff
@@ -77,6 +77,8 @@ const ROW_BODY_HEIGHT = 86
 const ROW_MARGIN_BOTTOM = 16
 export const ROW_SIZE = ROW_BODY_HEIGHT + ROW_MARGIN_BOTTOM
 
+const TRANSITION_PARAMS = '300ms ease-in-out'
```

Comment:
300ms feels a little slow in the demo when rapidly mousing over transactions, maybe something like 150ms?

Reply:
It was 300ms previously, but I can change it to 150.